README.md

```diff
@@ -184,6 +184,8 @@ the first available will be chosen, but you can force a specific one:
 let test#python#runner = 'pytest'
 " or
 let test#python#runner = 'nose'
+" or
+let test#python#runner = 'django'
```

Comment:
I've removed the reference. It's still slightly possible for it to be ambiguous if you, say, have the nose executable installed but are also testing a django app. But the django test runner is the default runner. So if you installed nose to test your django app with, this will currently do the "right" thing and find nose first.